Microsoft.Graph.sln

```diff
@@ -15,6 +15,10 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Microsoft.Graph.Core", "src
 EndProject
```

Comment:
Are you planning on having csproj and project.json? Can multitargeting be used here?